faasmcli/faasmcli/tasks/codegen.py

```diff
@@ -14,7 +13,7 @@
     join(FAASM_RUNTIME_ROOT, "lib", "fake", "libfakeLibB.so"),
 ]
 
-WAMR_WHITELISTED_FUNCS = [
+WAMR_ALLOWED_FUNCS = [
```

Comment:
Kind of unrelated, but following our `master->main` re-factor, I thought I would also change this whilst I was amending this file (I don't think we use whitelist/blacklist anywhere else other than sgx code which will get re-factored now anyway).

Reply:
Nice, yeah let's keep chipping away at this.